src/coffee/sync/utils/product.coffee

```diff
@@ -228,14 +249,14 @@ class ProductUtils extends BaseUtils
 
     _.sortBy actions, (a) -> a.action is 'addToCategory'
 
-  # Private: map product prices
+  # Map product prices
```

Comment:
Im not aware of these ones. What does this change do?

Reply:
just removes private flag from documentation I guess.. because we call this method from outside